.github/workflows/ci.yaml

```diff
@@ -90,6 +90,16 @@ jobs:
             echo Skipping unstable tests
             echo "mark_expression=not unstable" >> "$GITHUB_OUTPUT"
           fi
+      - name: Setup microceph
+        # setup microceph before tests
+        uses: phvalguima/microceph-action@v0.3.1
```

Comment:
may be we should add a TODO to use canonical action when it is available?

Reply:
Added as a task to @phvalguima under https://warthogs.atlassian.net/browse/DPE-2951